### ISSUE TITLE
Fails to build documentation

### ISSUE THREAD

**Issue by mcepl** [grantjenks/blue#26]
When building documentation for 0.5.1 (from the tarball from GitHub) for the package for openSUSE I got this failure:
```
[    4s] + cd docs
[    4s] + PYTHONPATH=../build/lib
[    4s] + make html
[    4s] Running Sphinx v3.3.1
[    4s] 
[    4s] Configuration error:
[    4s] There is a programmable error in your configuration file:
[    4s] 
[    4s] Traceback (most recent call last):
[    4s]   File "/usr/lib/python3.8/site-packages/sphinx/config.py", line 319, in eval_config_file
[    4s]     execfile_(filename, namespace)
[    4s]   File "/usr/lib/python3.8/site-packages/sphinx/util/pycompat.py", line 89, in execfile_
[    4s]     exec(code, _globals)
[    4s]   File "/home/abuild/rpmbuild/BUILD/blue-0.5.1/docs/conf.py", line 22, in <module>
[    4s]     project = blue.__title__.title()
[    4s] AttributeError: module 'blue' has no attribute '__title__'
[    4s] 
[    4s] make: *** [Makefile:20: html] Error 2
```

**Comment by grantjenks** [comment]
Oops, I broke that. I’m out now but can fix later today.

**Comment by grantjenks** [comment]
Reminder for me: add tox command for building the docs and running doc8

**Comment by grantjenks** [comment]
Fixed in ccf815d and deployed to PyPI as v0.5.2
